.husky/pre-commit

```diff
@@ -1,10 +1 @@
-branch="$(git rev-parse --abbrev-ref HEAD)"
```

Comment:
Don't delete or rename the husky files. Please restore these 3 files and package.json as they were.